### DISCUSSION TITLE
Using lifecycle hooks for array property

### DISCUSSION THREAD

**Discussion by rizkhal** [livewire/livewire#3262] [1. Help] [answered]
I have a form with clone function

I want to use the livewire lifecycle hook to populate some of the same fields.

This my form
![Screenshot from 2021-07-08 13-01-03](https://user-images.githubusercontent.com/24653114/124865063-93ce9f80-dfec-11eb-8b9a-079fb0565874.png)

This my component
![Screenshot from 2021-07-08 12-54-04](https://user-images.githubusercontent.com/24653114/124864836-2b7fbe00-dfec-11eb-94de-5b2361a6d762.png)

I want to make the `credit field` same as the `debet field`

How to use `updatedFields($value)` if the fields is array and value only return string ?

**Comment by joshhanley** [comment, marked as answer]
@rizkhal you can use a second property `$key` that gets passed to the `updatedFields` method. See my comment here https://github.com/livewire/livewire/discussions/3234#discussioncomment-967415

In future, please copy and paste your code and put it in triple backticks \``` so we can copy and paste you code, either to run it, or be able to change some of it to show you.

Hope this helps!

**Reply by rizkhal** [reply]
Thank you very much @joshhanley, in the future I will copy and paste, thank you.

**Reply by joshhanley** [reply]
@rizkhal no worries! Thanks, just means we can help you better 🙂
